README.md

```diff
@@ -52,16 +52,13 @@ The boiler plate code you would write is
 ```rust
 #![no_std]
 
-extern crate tock;
-
-fn main() {
+async fn main() {
   // Your code
 }
 ```
 If you want to use heap based allocation you will have to add
 ```rust
 #![feature(alloc)]
-extern crate alloc;
```

Comment:
Sure about this? Could you double-check?

Reply:
Fixed this. The `feature` is unnecessary as far as I can judge but the `extern crate` must stay, sorry!